highfidelity/hifi
# WL 21665 - Window.takeSecondaryCameraSnapshot() should emit a stillSnapshotTaken signal

> Window.takeSecondaryCameraSnapshot() should emit a stillSnapshotTaken signal
> 
> Refer to WindowScriptingInterface.h and takeSnapShot() and takeSecondaryCameraSnapshot()
> Worklist [21665](https://worklist.net/21665)

## Test Plan
1. Install and Run This PR
2. Enable Advanced/Developer Menus
3. Open the Console, and run: `Window.stillSnapshotTaken.connect(function(e) { print(e); });`
4. Now run `Window.takeSecondaryCameraSnapshot()` and confirm that the console prints the file path to the new Snapshot.
